src/shoobx/mocks3/models.py

```diff
@@ -168,6 +168,12 @@ def etag(self):
                 self._etag = file_hash.hexdigest()
         return f'"{self._etag}"'
 
+    @property
+    def size(self):
+        with open(self._value_path, "rb") as file:
+            file.seek(0, 2)
```

Comment:
Is that really needed? A freshly opened file should be at position 0.

Reply:
You might also be able to use the file size attribute, if the path is always a filesystem path. (Or have a condition.)